lib/govuk_publishing_components/presenters/highlight_boxes_helper.rb

```diff
@@ -0,0 +1,17 @@
+module GovukPublishingComponents
+  module Presenters
+    class HighlightBoxesHelper
+      def initialize(options)
+        @items = options[:items]
+      end
+
+      def data_tracking?
+        @items.each do |item|
```

Comment:
Minor: you could use `@items.any? { |item| item[:link][:data_attributes] }` to achieve this.